sdk/storage/storage-queue/README.md

```diff
@@ -198,27 +296,63 @@ Messages are processed in two steps.
 If your code fails to process a message due to hardware or software failure, this two-step process ensures that another instance of your code can get the same message and try again.
 
 ```javascript
-const response = await queueClient.receiveMessages();
-if (response.receivedMessageItems.length == 1) {
-  const receivedMessageItem = response.receivedMessageItems[0];
-  console.log(`Processing & deleting message with content: ${receivedMessageItem.messageText}`);
-  const deleteMessageResponse = await queueClient.deleteMessage(
-    receivedMessageItem.messageId,
-    receivedMessageItem.popReceipt
-  );
-  console.log(
-    `Delete message succesfully, service assigned request Id: ${deleteMessageResponse.requestId}`
-  );
+const { DefaultAzureCredential } = require("@azure/identity");
+const { QueueServiceClient } = require("@azure/storage-queue");
+
+const account = "<account>";
+const credential = new DefaultAzureCredential();
+
+const queueServiceClient = new QueueServiceClient(
+  `https://${account}.queue.core.windows.net`,
+  credential
+);
+
+const queueName = "aNewQueue";
+
+async function main() {
+  const queueClient = queueServiceClient.getQueueClient(queueName);
+  const response = await queueClient.receiveMessages();
+  if (response.receivedMessageItems.length == 1) {
+    const receivedMessageItem = response.receivedMessageItems[0];
+    console.log(`Processing & deleting message with content: ${receivedMessageItem.messageText}`);
+    const deleteMessageResponse = await queueClient.deleteMessage(
+      receivedMessageItem.messageId,
+      receivedMessageItem.popReceipt
+    );
+    console.log(
+      `Delete message succesfully, service assigned request Id: ${deleteMessageResponse.requestId}`
+    );
+  }
 }
+
+main();
 ```
 
 ### Delete a queue
 
 ```javascript
-const deleteQueueResponse = await queueClient.delete();
-console.log(
-  `Deleted queue successfully, service assigned request Id: ${deleteQueueResponse.requestId}`
+const { DefaultAzureCredential } = require("@azure/identity");
+const { QueueServiceClient } = require("@azure/storage-queue");
+
+const account = "<account>";
+const credential = new DefaultAzureCredential();
+
+const queueServiceClient = new QueueServiceClient(
+  `https://${account}.queue.core.windows.net`,
+  credential
 );
+
+const queueName = "aNewQueue";
+
+async function main() {
+  const queueClient = queueServiceClient.getQueueClient(queueName);
+  const deleteQueueResponse = await queueClient.delete();
+  console.log(
+    `Deleted queue successfully, service assigned request Id: ${deleteQueueResponse.requestId}`
+  );
+}
+
+main();
```

Comment:
Doesn't it seem to be an overkill to have too much repetitive code in the snippets?

@ramya-rao-a

Reply:
@HarshaNalluru are you referring to the repeated requires and,  QueueServiceClient instantiation? I personally find it useful to have each snippet working by itself and don't mind the repetition. That way I don't have to move back and forth to find and fill out the missing parts, especially if not following the snippets from the beginning

Reply:
Fine if people like it :)

Reply:
I think we should lean towards samples in the READMEs being runnable. This is what I've done in KeyVault as well so I think we should keep it this way. We can always consolidate/simplify later.